canvas_admin_tools/templates/canvas_admin_tools/dashboard.html

```diff
@@ -0,0 +1,73 @@
+{% extends 'canvas_admin_tools/base.html' %}
+
+{% load static %}
+
+{% block stylesheets %}
+<link href="{% static 'canvas_admin_tools/css/dashboard.css' %}" rel="stylesheet" />
+{% endblock stylesheets %}
+
+{% block content %}
+<header>
+    <div class="row lti-header">
+        <h1>Administration Tasks</h1>
+    </div>
+</header>
+
+<main>
+    {% if has_manage_courses %}
+    <div class="col-md-6">
+        <div class="panel panel-default">
+            <div class="panel-body">
+                <h3>Manage Courses</h3>
+                <div class="list-group">
+                    {% if canvas_site_creator %}
+                    <a href="{{ canvas_site_creator }}" class="list-group-item active" target="_parent">
+                        <h4 class="list-group-item-heading">Canvas Site Creator</h4>
+                        <p class="list-group-item-text">Create Canvas sites for one or multiple courses</p>
+                    </a>
+                    {% endif %}
+                </div>
+            </div>
+        </div>
+    </div>
+    {% endif %}
+    {% if has_manage_account %}
+    <div class="col-md-6">
+        <div class="panel panel-default">
+            <div class="panel-body">
+                <h3>Manage My Account</h3>
+                <div class="list-group">
+                    {% if conclude_courses %}
+                    <a href="{{ conclude_courses }}" class="list-group-item active" target="_blank">
+                        <h4 class="list-group-item-heading">Conclude Courses</h4>
```

Comment:
Why is the target _blank for this one while the rest are opening in the parent frame?